spec/requests/restaurants_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+RSpec.describe 'Restaurants API', type: :request do
+  # initialize test data
+  let!(:restaurants) { create_list(:restaurant, 10) }
+  let(:restaurant_id) { restaurants.first.id }
+
+  # Test suite for GET /restaurants
+  describe 'GET /restaurants' do
+    # make HTTP get request before each example
```

Comment:
No need for such a comment, the before already replies that get request is done before each example